blog/poc3.md

```diff
@@ -0,0 +1,120 @@
+A [*manifest*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Manifests) is a data structure describing file collections; they specify paths and corresponding content hashes allowing for URL-based content retrieval. The `bzz` URL scheme assumes that the content referenced in the domain is a manifest and renders the content entry whose path matches the one in the request path. This offers the functionality of virtual hosting of websites without dedicated webservers. Hosting on web3 is thus analogous to web2.0, with centralised hosting taken out of the equation.
+Manifests can also be mapped to a filesystem directory tree, which allows for uploading and downloading directories. Finally, manifests can also be considered indexes, so it can be used to implement a simple key-value store, or alternatively, a database index.
+
+In sum, the overall vision of Swarm offers a fully decentralised peer-to-peer internet which is not only fault-tolerant, has zero downtime and offers censorship resistance but is also economically self-sustaining due to a built-in incentive system. By compensating nodes for contributing their bandwidth and disk space, these incentives achieve reliable low-latency scalable retrieval of popular content on the one hand and guarantees persistence of important yet rarely accessed data like archives or backups on the other. For smooth delivery Swarm will use the SWAP protocol while for storage guarantees it will use a two-tiered insurance system.
+
+# The year behind
+
+In the past year [the Swarm team](https://pbs.twimg.com/media/DetPkqZX0AAAsPp.jpg:large) has grown in size and now on fire delivering the vision. We have been busy redesigning the network layer, rewriting the retrieval protocol using a stream abstraction, rewriting connectivity management and the overlay network code as well as developed a very sophisticated *network simulation framework*.
+This latter feature is a main differentiator of the project and allows us to trial very complex emergent properties of a network of nodes, test algorithmic correctness, fault tolerance and scalability. POC3 code was finalised just for the [Swarm Orange Summit in Ljubljana](https://swarm-gateways.net/bzz:/2018.swarm-summit.eth), where we had 80 participants and a very inspiring and creative week ([watch this two-minute video hosted on Swarm] (https://swarm-gateways.net/bzz:/079b4f4155d7e8b5ee76e8dd4e1a6a69c5b483d499654f03d0b3c588571d6be9/)) of talks and coding. It is inspiring to see a growing number of  [contributors](https://github.com/ethersphere/go-ethereum/blob/b14d635539a7fd548bd1fe4fe987f137229ff38e/swarm/AUTHORS) and [companies that want to build on swarm](http://swarm-guide.readthedocs.io/en/latest/introduction.html#Sponsors)
```

Comment:
Let's start a new paragraph with the sentence "POC3 code was....",and the first sentence about the simulation can go to the previous paragraph.